Sources/FacebookRemoteCommand.swift

```diff
@@ -38,6 +38,7 @@ public class FacebookRemoteCommand: RemoteCommand {
     }
 
     func processRemoteCommand(with payload: [String: Any]) {
+        facebookInstance?.checkAdvertiserTracking()
```

Comment:
I know you didn't add this, but why is `facebookInstance` an optional var instead of a non optional let? I think it would be nice if we changed it.

Reply:
Im not sure what all this would effect with current integrations so im going to leave it as an optional for now.